Bottom: 14ebf5701bba2dc96fb1f29b02dee56748353c6f
Top:    faf3bcc67324b9045716085982dc7bff2e93e4aa
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-05-09 16:22:42 +0800

Refresh of macvlan-introduce-flag-to

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index dad2936..68b2963 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -504,7 +504,7 @@ static int macvlan_queue_xmit(struct sk_buff *skb, struct net_device *dev)
 xmit_world:
 	skb->dev = vlan->lowerdev;
 	if (vlan->flags & MACVLAN_FLAG_DIRECT)
-		return skb_direct_xmit(skb);
+		return skb_direct_xmit(skb, false);
 	else
 		return dev_queue_xmit(skb);
 }
